fgd/brush/trigger/trigger_momentum_timer_start.fgd

```diff
@@ -1,16 +1,12 @@
-@SolidClass base(ZoneTrigger)
-	appliesto(MOMENTUM)
-= trigger_momentum_timer_start : "Trigger that starts the timer." 
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_start : "Zone that starts the timer." 
 	[
-	speed_limit(float) : "Speed limit (UPS)" : 350 : "Max speed (Units Per Second) the player will be able to have, depending on the limit speed type."
-	look_angles(angle) : "Look angles" : "0 0 0" : "Where player will look when teleported to start."
-	start_on_jump(integer) : "Start timer on jump" : 1 : "Start the timer when the player jump in the start zone/trigger zone."
-	speed_limit_type(integer) : "Limit speed type" : 0 : "0) Limit the speed of the player no matter what. 1) Limit the speed only in air. 2) Limit the speed only on ground. 3) Limit the speed only on landing."
-	teleport_destination(target_destination) : "Remote Destination" : : "The entity specifying the point to which the player should be teleported when restarting a run."
-
-	spawnflags(flags) =
-		[
-		8192: "Limit leave speed" : 1
-		16384: "Use look angles" : 0
-		]
+	targetname(target_source) : "Name" : : "The name of this zone."
+	filtername(string) : "Filter Name" : : "filter_activator entity to use for this zone."
+	track_number(integer) : "Track Number" : 0 : "Track number that this entity belongs to. 0 = main map, 1+ = bonus zone (of that number)."
+	segments_ordered(boolean) : "Segments Ordered" : 1 : "Do the stages/checkpoints need to be completed in order?"
```

Comment:
```suggestion
	segments_ordered(boolean) : "Segments Ordered" : 1 : "Do the stages need to be completed in order? If not, every zone must also have an associated trigger_momentum_timer_stage_end."
```

Reply:
Does this only apply to stages? I can rename this to stages_ordered if so.

Reply:
Yeah that would make sense. I would change the display name to "Stages Ordered" for clarity but maybe keep the key as `segments_ordered` since that's the internal term.